min.py

```diff
@@ -146,9 +178,28 @@ def get_min_scipy_version(min_python, min_numpy):
         MIN_PYTHON = get_min_python_version()
     MIN_NUMBA, MIN_NUMPY = get_min_numba_numpy_version(MIN_PYTHON)
     MIN_SCIPY = get_min_scipy_version(MIN_PYTHON, MIN_NUMPY)
+
     print(
         f"python: {MIN_PYTHON}\n"
         f"numba: {MIN_NUMBA}\n"
         f"numpy: {MIN_NUMPY}\n"
         f"scipy: {MIN_SCIPY}"
     )
+
+    pkgs = {
+        "numpy": "1.19",
```

Comment:
Should this be `MIN_NUMPY` instead of `1.19`?

Reply:
Yes! That's the result of staying up too late 🤦